functional_tests.go

```diff
@@ -1525,11 +1528,6 @@ func testFPutObject() {
 		return
 	}
 
-	err = os.Remove(fName + ".gtar")
```

Comment:
This file should be removed. This could is needed, otherwise it will leave an extra file in the /mint/data directory